charts/fybrik/values.yaml

```diff
@@ -296,14 +327,24 @@ opaServer:
   # the equivalent fields of .Values.global.podSecurityContext.
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
   podSecurityContext: {}
+  # for tls support add:
+  #  runAsNonRoot: false
+  #  runAsUser: 0
 
   # Container Security Context. If set, the fields of containerSecurityContext override
   # the equivalent fields of .Values.global.containerSecurityContext.
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
   containerSecurityContext: {}
+  # for tls support add:
```

Comment:
lines 338-343 ?

Reply:
same as above. without this opa server failed with error:
```
{"err":"listen tcp 0.0.0.0:443: bind: permission denied","level":"error","msg":"Listener failed.","time":"2022-08-08T04:47:08Z"}

Reply:
of course, ports below 1024 require root permissions, this is the reason to use 8xxx ports

Reply:
I changed opa server port to 8443